ui/src/shared/components/TimeMachine/TimeMachine.tsx

```diff
@@ -538,20 +536,13 @@ class TimeMachine extends PureComponent<Props, State> {
     this.setState({isViewingRawData: !this.state.isViewingRawData})
   }
 
-  private toggleFluxOn = (): void => {
-    if (!this.isFluxSelected) {
-      const newQueryType = QueryType.Flux
-      const source = this.useDynamicSource ? null : this.source
-      this.updateQueryDraftsSource(source, newQueryType)
-    }
-  }
-
-  private toggleFluxOff = (): void => {
-    if (this.isFluxSelected) {
-      const newQueryType = QueryType.InfluxQL
-      const source = this.useDynamicSource ? null : this.source
-      this.updateQueryDraftsSource(source, newQueryType)
-      this.setState({isViewingRawData: false})
```

Comment:
This `setState` fixed a crash resulting from switching between query types in the Data Explorer. Does removing it reintroduce the bug?

https://github.com/influxdata/chronograf/issues/4618
https://github.com/influxdata/chronograf/pull/4658/commits/6700c597c0276aa4c235a40c7a76ebfc2502b779